CHANGELOG.md

```diff
@@ -2,10 +2,33 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
-# 10.0.0-rc.1 June 10, 2021
+# 10.0.0-rc.2 June 23, 2021
 
 **The Mapbox Maps SDK for Android has moved to release candidate status and is now ready for production use.**
 
+## Features ✨ and improvements 🏁
+* Add Persistent Layer API so that the tagged layer and its associated resources would remain when a style is reloaded. This improves performance of Annotation and Location Component Plugin during the style change. ([#368](https://github.com/mapbox/mapbox-maps-android/pull/368))
```

Comment:
We also need to mention the specific experimental APIs we introduced:
`Introduce experimental Style#addPersistentStyleLayer, Style#addPersistentStyleCustomLayer and Style#isStyleLayerPersistent APIs.` in https://github.com/mapbox/mapbox-maps-android/pull/422
and 
```
@MapboxExperimental
Style.addPersistentLayer(layer: Layer, position: LayerPosition? = null) {
  layer.bindPersistentlyTo(this, position)
}
@MapboxExperimental
fun Layer.isPersistent(): Boolean? 
```
in https://github.com/mapbox/mapbox-maps-android/pull/368

Reply:
> Can we include changes from internal upstream dependencies?
> 
> edit: I guess these are the ones listed without a link?

Yes, but missing entries for core and telem bump, will add shortly.